applications/openshift/etcd/etcd_auto_tls/oval/oval.xml

```diff
@@ -0,0 +1,32 @@
+<def-group>
+  <definition class="compliance" id="etcd_auto_tls" version="1">
+    <metadata>
+      <title>Disable etcd Self-Signed Certificates</title>
+      <affected family="unix">
+        <platform>multi_platform_ocp</platform>
```

Comment:
Wasn't sure how to build out the platform. Thinking:
- Add Operating Systems OCP is certified to run on (rhel7)
- Need new CPE for OCP3 itself

Perhaps multi_platform_ocp can capture both of the above? What do you think?